### DISCUSSION TITLE
radio player not displaying on https site

### DISCUSSION THREAD

**Discussion by gulicny** [AzuraCast/AzuraCast#6057] [General]
Hey y'all I am new with Azuracast so I appreciate your patience! A couple questions: 

I am trying to embed the radio player on my site but after updating the site to HTTPS the iframe link is no longer displaying. I have attempted to update the iframe link to https but no luck. 

I am also unable to use the css editor to style the player for some reason (the text is displaying as white/grey no matter which theme i choose.)

Any help would be very appreciated!

**Comment by BusterNeece** [comment]
@gulicny If you visit the embed URL directly via HTTPS in your browser (i.e. in a full tab and not just in the iframe), do you see any more information on the error? There's lots of things that could cause problems there, but if both the host site and the AzuraCast installation have active SSL certificates (i.e. via LetsEncrypt) things should work just fine.

**Reply by gulicny** [reply]
I am able to see this no problem using the HTTP link so I suspect the issue may be that I have not set up SSL for the Azuracast installation. Will Get on that now! Thank you. 

Still unable to get the "dark" theme to display properly in my browser for some reason....

**Comment by Piratenkanon** [comment]
you must activate the dark theme in system settings and also in your profile settings
